Functions.Templates/Templates/EventGridBlobTrigger-Python/__init__.py

```diff
@@ -0,0 +1,9 @@
+import logging
```

Comment:
add the template to `/Build/PackageFiles/ExtensionBundle/ExtensionBundleTemplates-4.x.nuspec`

Reply:
Added